src/blocks/Blocks.astro

```diff
@@ -1,5 +1,6 @@
 ---
 import type { AnyBlock } from './Blocks';
+import DetailsBlock from './DetailsBlock/DetailsBlock.astro';
```

Comment:
About the name of the component. I get your feedback on how it doesn't match with the HTML element. But I think that's an implementation detail. Clients also call things you click on buttons while in HTML those may be anchors. To me it's important that the name of the component reflects its purpose. The aim of this component is to structure things like product details. Hence the Details Block.